homeassistant/components/netatmo/config_flow.py

```diff
@@ -45,11 +65,113 @@ def extra_authorize_data(self) -> dict:
 
     async def async_step_user(self, user_input=None):
         """Handle a flow start."""
-        if self.hass.config_entries.async_entries(DOMAIN):
-            return self.async_abort(reason="already_setup")
-
+        await self.async_set_unique_id(DOMAIN)
```

Comment:
Why don't we use the username or account ID?

Reply:
Simply because it was never intended to support more than one account. But there is no reason not to.